.gitattributes

```diff
@@ -0,0 +1,63 @@
+###############################################################################
```

Comment:
How are you submitting this Pull Request? Is it through Visual Studio or some other IDE? It's possible this file is being generated by whichever IDE you're using; merging this in to the root of the repo will affect everyone, so I can't merge this file in...